Makefile

```diff
@@ -7,7 +7,9 @@ INSTALL ?= install
 SASSC ?= sassc
 MKDIR ?= mkdir
 CONVERT ?= convert
-
+WEBPACK_DEV_SERVER := ./node_modules/webpack-dev-server/bin/webpack-dev-server.js
+WEBPACK_CONFIG := ./www/webpack.config.dev.js
```

Comment:
This one is being used in a single place only, so you can inline it there instead